Tasks/CondaEnvironmentV1/toolutil.ts

```diff
@@ -0,0 +1,20 @@
+import * as path from 'path';
+
+import * as task from 'vsts-task-lib/task';
+
+/**
+ * Like `prependPath` from vsts-task-tool-lib, but does not check whether the directory exists.
+ * TODO move this to vsts-task-tool-lib
+ */
+export function prependPathSafe(toolPath: string) {
+    // TODO task-lib 2.4.0: `assertAgent` is not in mock-task
```

Comment:
I've worked with Talos on a plan for task-lib that will fix stuff like this.  (Right now you can't mix custom mocks with `setAnswers` on mock-task.)  I hope to get to that later this sprint.